Sources/Bot/Services/MergeService.swift

```diff
@@ -565,19 +586,8 @@ extension MergeService {
         return Feedback(predicate: { $0.status != .starting }) { state in
             return pullRequestChanges
                 .observe(on: scheduler)
-                .map { metadata, action -> Event.Outcome? in
-                    switch action {
-                    case .opened where metadata.reference.isLabelled(with: state.integrationLabel):
-                        return Event.Outcome.include(metadata.reference)
-                    case .labeled where metadata.reference.isLabelled(with: state.integrationLabel) && metadata.isMerged == false:
-                        return Event.Outcome.include(metadata.reference)
-                    case .unlabeled where metadata.reference.isLabelled(with: state.integrationLabel) == false:
-                        return Event.Outcome.exclude(metadata.reference)
-                    case .closed:
-                        return Event.Outcome.exclude(metadata.reference)
-                    default:
-                        return nil
-                    }
+                .map { metadata, action in
```

Comment:
`map`+`skipNil` can be flatten using `filterMap`.

Reply:
Good point, not sure why we didn't spot it earlier 👍